var/spack/repos/builtin/packages/py-torch/package.py

```diff
@@ -419,6 +419,13 @@ class PyTorch(PythonPackage, CudaPackage, ROCmPackage):
         when="@2.0.0:2.0.1",
     )
 
+    # Use correct OpenBLAS include path under prefix
+    patch(
+        "https://patch-diff.githubusercontent.com/raw/pytorch/pytorch/pull/110063.patch?full_index=1",
+        sha256="23fb4009f7337051fc5303927ff977186a5af960245e7212895406477d8b2f66",
+        when="@:2.1",
```

Comment:
@adamjstewart I believe this patch is needed for `@2.1.1` as well. I'm hitting this using py-torch with openblas.

```
  -- Trying to find preferred BLAS backend of choice: Generic
  -- Using BLAS: BLAS_LIBRARIES-NOTFOUND
```

Reply:
The patch only applied once py-torch hit v2.2. So 2.1.1 and 2.1.2 would still need the patch

https://github.com/pytorch/pytorch/commit/21d77bcf808d076f81b5e885a8ce6ca20a08dbed

Reply:
`@:2.1` includes 2.1.X